snapcraft/internal/build_providers/_snap.py

```diff
@@ -46,11 +46,18 @@ class _SnapOp(enum.Enum):
 
 class _SnapManager:
     def __init__(
-        self, *, snap_name: str, snap_dir: str, latest_revision: str, snap_arch: str
+        self,
+        *,
+        snap_name: str,
+        snap_dir: str,
+        latest_revision: str,
+        snap_arch: str,
+        store_only: bool
```

Comment:
The name of this parameter (both here and in `SnapInjector.add()`) does not really indicate what it does. It's literally saying "don't inject this snap from the host", right? Perhaps `inject_from_host: bool` would better indicate what it does?

Reply:
I'll switch to this naming convention